datafusion/src/physical_plan/coercion_rule/aggregate_rule.rs

```diff
@@ -154,6 +155,15 @@ pub(crate) fn coerce_types(
             }
             Ok(input_types.to_vec())
         }
+        AggregateFunction::ApproxMedian => {
+            if !is_approx_median_support_arg_type(&input_types[0]) {
```

Comment:
Perhaps this could use `is_approx_percentile_cont_supported_arg_type` to ensure it doesn't ever drift from the underlying implementation and remove duplication?

Reply:
Good point.